### ISSUE TITLE
What are the statistics affected by `alfred qc --bed`?

### ISSUE THREAD

**Issue by evanbiederstedt** [tobiasrausch/alfred#17]
Hi Tobias

I haven't dug closely into the source code for this, so apologies if this question is a bit lazy:

What are the metrics affects by using the optional `--bed` flag for `alfred qc`? 

I suspect this affects things like target coverage calculated....but I'm not sure what else. 

Given a standard WGS normal BAM at 40x, what would you expect the different to be between including the target BED or excluding it? 

Thank you for the help

**Comment by tobiasrausch** [comment]
Hi,

The optional BED file of target regions will not affect the whole-genome statistics. For the [Alfred web app](https://gear.embl.de/alfred/) you will get the same statistics in the "Summary stats" tab and in addition some summary statistics for the BED file like the fraction of reads in a BED target region.

For the GC content, you will still have the sample and reference GC distribution but in addition the GC distribution for the BED file.

Most importantly you will get 2 additional plots related to the input BED file:
(1) Target coverage distribution (Fraction of targets above a coverage level 1x, 2x, 3x, ....)
(2) The on-target rate (Fraction of reads on target at 0bp extension, 25bp extension, ...)

In short, a BED file makes sense for all targeted assays (whole-exome sequencing, Haloplex, PCR amplicon sequencing, ...)

Best, Tobias

**Comment by evanbiederstedt** [comment]
Hi @tobiasrausch

Thank you much for the prompt reply. 

> For the GC content, you will still have the sample and reference GC distribution but in addition the GC distribution for the BED file.

> Most importantly you will get 2 additional plots related to the input BED file:
(1) Target coverage distribution (Fraction of targets above a coverage level 1x, 2x, 3x, ....)
(2) The on-target rate (Fraction of reads on target at 0bp extension, 25bp extension, ...)

Ah, this does make a great deal of sense. Right, I see how this works now in the source code a bit better now as well. 

Thank you for the help! I'm still not entirely sure how this relates to tools for targeted assays which require both target and baits intervals.

**Comment by evanbiederstedt** [comment]
Hi @tobiasrausch

Thanks for the prompt reply.
